test/functional/ex_cmds/map_spec.lua

```diff
@@ -26,3 +27,131 @@ describe(':*map', function()
     expect('-foo-')
   end)
 end)
+
+describe(':*map <expr>', function()
+  before_each(function()
```

Comment:
add `local screen` above this line to fix these lint errors:

    test/functional/ex_cmds/map_spec.lua:34:5: setting non-standard global variable screen
    test/functional/ex_cmds/map_spec.lua:35:5: accessing undefined variable screen

The linter is noticing that we didn't define `screen`.